src/components/autocomplete/autocomplete.stories.js

```diff
@@ -0,0 +1,140 @@
+import { html } from "lit-html";
+import "../autocomplete";
+import { docsIntro } from "../../../.storybook/utils.js";
+
+export default {
+  title: 'custom-elements/vl-autocomplete',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'autocomplete',
+          intro: 'Use autocomplete to show ...',
+        }),
+      },
+    },
+  },
+};
+
+//--------------------------
+
+export const Default = () => html`
+  <vl-autocomplete data-min-chars="1" 
+                   data-static-list='[
```

Comment:
De data-static-list array komt hier en daar voor in de code. Misschien maken we hier beter een variable van die we kunnen hergebruiken.

Reply:
Dat klopt inderdaad maar ik zie niet goed hoe je die variable in die stories.js dan meegeeft aan het de specifieke stories. Heb je daar misschien een voorbeeld van?